src/quoridor/controller/gamecontroller.go

```diff
@@ -5,50 +5,99 @@ import (
 
 	"quoridor/game"
 	"quoridor/storage"
-
-	"github.com/lithammer/shortuuid"
 )
 
+const (
+	UnknownPlayer = 0
+	FirstPlayer = 1
+) 
+
+type Party struct {
+	game game.Game
+	players map[string]int
+}
+
+func (p Party) isReady() bool {
+	return len(p.players) == len(p.game.Pawns)
+}
+
+func getParty(id string) (Party, error) {
+	p, found := storage.Get(id)
+	if !found {
+		return Party{}, errors.New("The game does not exist")
+	}
+	return p.(Party), nil
+}
+
+func checkPlayerCanPlay(p Party, playerID string) error {
+	player := p.players[playerID]
+	if player == UnknownPlayer {
+		return errors.New("Forbidden")
+	}
+	if !p.isReady() {
+		return errors.New("Game is not ready")
+	}
+	if player != p.game.PlayerTurn {
+		return errors.New("It is not your turn")
+	}
+	return nil
+}
+
 // CreateGame create a game with the default configuration
-func CreateGame(conf *game.Configuration) (*game.Game, error) {
-	boardSize := conf.BoardSize
-	board, err := game.NewBoard(boardSize)
+func CreateGame(conf game.Configuration, playerID string) (*game.Game, error) {
```

Comment:
As previously said, you don't need to pass the playerId here because there's no player at game init